### PR TITLE
Move to material design

### PR DESCRIPTION
**Pull request by vidya-ram**
Baseframe dependency https://github.com/hasgeek/baseframe/pull/263

### CODE REVIEW

**Comment by jace** [comment]
@vidya-ram Third party JS should be moved into a subfolder. It's hard to tell which bits are ours otherwise.

File: imgee/templates/delete.html.jinja2 [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 	{% endblock %}
-	<div class="alert alert-error">
-		<h5><i class="icon-trash"></i>&nbsp;&nbsp;Confirm delete</h5><hr />
-		<p>Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
+	<div class="alert alert--error">
+		<p class="mui--text-title">{% assets "fa5-sprite" %}<svg class="fa5-icon fa5--align-baseline fa5-icon--subhead " aria-hidden="true" role="img"><use xlink:href="{{ ASSET_URL }}#trash-alt"></use></svg>{% endassets %} Confirm delete</p><hr/>
+		<p class="mui--text-subhead">Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
 		<p id="warning">
-			<i class="icon-warning-sign"></i>&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
+			{% assets "fa5-sprite" %}<svg class="fa5-icon fa5--align-baseline" aria-hidden="true" role="img"><use xlink:href="{{ ASSET_URL }}#info-circle"></use></svg>{% endassets %}&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
```
Comment (jace) [inline imgee/templates/delete.html.jinja2]: Many templates have mixed tabs and spaces. `pre-commit` and linters are needed here.

File: imgee/templates/index.html.jinja2 [hunk 2]
```diff
@@ -1,35 +1,25 @@
 {% extends "layout.html.jinja2" %}
-{% block titletags %}
-  <title>{% block title %}{{ config['SITE_TITLE'] }}{% endblock %}</title>
```
Comment (jace) [inline imgee/templates/index.html.jinja2]: We should drop use of `config['SITE_TITLE']` now as it interferes with i18n. This code is no longer meant for others to run their own installations.

**Comment by vidya-ram** [comment]
> @vidya-ram Third party JS should be moved into a subfolder. It's hard to tell which bits are ours otherwise.

I had removed all third party JS to baseframe, missed deleting jquery.jeditable.js. Fixed it now.

**Pull request merged**